ludwig/hyperopt/run.py

```diff
@@ -7,6 +7,7 @@
 
 import pandas as pd
 import yaml
+from ray.tune import Callback as TuneCallback
```

Comment:
@jeffkinnison We'll want to move this into the try-except block on line 46, and then set `TuneCallback` to None or something like that. The reason is because Ray/Dask falls under "distributed" tests, so this will cause failures in the non-distributed tests when collecting all the tests to run

Reply:
@jeffkinnison I think you'll still need to address this to prevent the test failures in the non-distributed tests

Reply:
I removed it entirely since it was only used for type hints. Replaced with a string in [a073cc0](https://github.com/ludwig-ai/ludwig/pull/2640/commits/a073cc00aec3e3915c4cdd143aabbee7a247c816)